modules/administration-guide/partials/proc_configuring_bitbucket_authentication.adoc

```diff
@@ -0,0 +1,52 @@
+// configuring-bitbucket-authentication
+
+[id="configuring_bitbucket_authentication_{context}"]
+= Authentication on Bitbucket servers
+
+{prod} users may use public or private Bitbucket Git repositories as a source of theirs projects. The standard
+factory flow using devfile at the root of the repository is available starting of {version} of {prod}.
+
+Using private repositories, requires some additional configuration described below.
+
+Bitbucket authentication is based on using so-called "personal access tokens". Each Bitbucket user is able to request some
+amount of personal access tokens with different names, permissions, expiration times and so on. Those tokens
+may be used to sign Bitbucket REST API calls and perform Git repository operations.
+
+To allow Bitbucket authentication on {prod} side, personal tokens must be stored in user's namespace in a form of
```

Comment:
Not sure if users can  create secret with cretential for themselves ?

Reply:
Do you mean that RBAC don't allow them to create secrets?
I check and in case of OSIO, users are not able to CRUD secrets in che related namespaces(suffixed with -che), but they are able in their personal.

But at the same time is admin really is supposed to create those secrets? I don't think SCM admin has access to other users personal tokens.
Is no - who is supposed to create these secrets?

Reply:
I believe this feature will never be enabled on OSIO, it's purely for custom CRW installations. And i don't know if the CRW users is able to access to something more than theirs UD (i mean K8S/Openshift console/CLI/etc) to manage their own namespaces and create secrets in them. It's looks like more admin stuff as for me.

Reply:
I agree with @sleshchenko, how an admin is supposed to get the token of another user? I don't think that's possible. And as for other secrets in the user namespace, those can/should be managed by the user. Ideally users should only see UD, that's why we are automating the creation of those secrets, but we are not there yet.

Reply:
So how the users suppored to create secrets for themself ?

Reply:
We have a full documentation section for that: https://www.eclipse.org/che/docs/che-7/end-user-guide/mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container/. That's where I was expecting to find the documentation for the bitbucket token too.

I agree that all those secrets should be managed from the UD, not manually with `kubectl`. We have [an issue for that](https://github.com/eclipse/che/issues/17954). But we are not there yet and anyway those should not be included among admin tasks (admins do not manage users git token, ssh keypairs etc...)